src/Connectors/src/Connector/ConfigurationExtensions.cs

```diff
@@ -161,6 +161,20 @@ public static bool HasCloudFoundryServiceConfigurations(this IConfiguration conf
         return configuration.GetSection("vcap:services").GetChildren().Any();
     }
 
+    /// <summary>
+    /// Evaluate whether an IConfiguration contains Kubernetes service bindings.
+    /// </summary>
+    /// <param name="configuration">
+    /// Application Configuration.
+    /// </param>
+    /// <returns>
+    /// true if k8s:bindings found in configuration, otherwise false.
+    /// </returns>
+    public static bool HasCloudNativeBindings(this IConfiguration configuration)
```

Comment:
Should this be named HasKubernetesServiceBindings? Because we have the extension method `AddKubernetesServiceBindings`. I wonder why we use different terminology here, and how they relate to each other.

Reply:
For consistency within Steeltoe, you're right. They are the same thing, I just tend to refer to them as CNB in my head and it leaked out here